src/components/Team/CardContainer.jsx

```diff
@@ -0,0 +1,272 @@
+import React from 'react';
+import "../../utility/fonts.css";
+import {StaticP, StaticH1} from "../../utility/ContentStyles.js";
+import './CardContainer.css';
+import MemberCard from './MemberCard.jsx';
+
+
+class CardContainer extends React.Component {
+
+    constructor(props){
+        super(props);
+    }
+
+    render(){
+        return(
+            <React.Fragment>
+                <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"></link>
+                <StaticH1>
+                    Executive Team
+                </StaticH1>
+                <div className="centered">
+                    <section className="cards">
+                      <div className="cardContainerColumn">
+                          <MemberCard name="Benjamin Carney" title="President"
+                            gradYear="2020"
+                            interests="Fitness, trail running, nutrition, playing music"
+                            funFact="Voluntarily takes cold showers on a daily basis and only eats plants 🌱 "
```

Comment:
Perhaps we could use default props rather than manually setting these to be the empty string